### PR TITLE
Implemented code for static UI of Settings page (#202re98)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

**Comment by azkyakhan** [comment]
settings-desktop

settings mobile

**Comment by azkyakhan** [comment]
https://user-images.githubusercontent.com/82817616/154490335-df30b99a-2ccb-49b3-bd00-abc98f6dfd56.mp4

File: src/views/Settings.vue [hunk 1]
```diff
@@ -2,60 +2,123 @@
   <ion-page>
     <ion-header :translucent="true">
       <ion-toolbar>
+        <ion-menu-button slot="start" />
```
Comment (dt2patel) [inline src/views/Settings.vue]: Should we remove the menu button too?

File: src/views/Settings.vue [hunk 2]
```diff
@@ -2,60 +2,123 @@
...
         <ion-back-button slot="start" default-href="/" />
         <ion-title>{{ $t("Settings") }}</ion-title>
       </ion-toolbar>
     </ion-header>
-    
+
     <ion-content :fullscreen="true">
```
Comment (dt2patel) [inline src/views/Settings.vue]: Remove full screen. We shouldn't use full screen unless you have to for a UI reason.

File: src/views/Settings.vue [hunk 3]
```diff
@@ -2,60 +2,123 @@
...
-      <!-- Select store -->
-      <ion-item>
-        <ion-icon :icon="storefrontOutline" slot="start" />
-        <ion-label>{{ currentFacility.facilityId ? currentFacility.facilityId : '' }}</ion-label>
-        <ion-select interface="popover" :value="currentFacility.facilityId" @ionChange="setFacility($event)">
-          <ion-select-option v-for="facility in userProfile.facilities" :key="facility.facilityId" :value="facility.facilityId" >{{ facility.facilityId }}</ion-select-option>
-        </ion-select>
-      </ion-item>
+      <div>
```
Comment (dt2patel) [inline src/views/Settings.vue]: Use main tag

File: src/views/Settings.vue [hunk 4]
```diff
@@ -2,60 +2,123 @@
...
+          <ion-item lines="none">
+            <ion-icon :icon="personCircleOutline" slot="start" />
+            <ion-label>{{ $t("Profile") }}</ion-label>
+          </ion-item>
+          <ion-item>
+            <ion-label>{{ $t("User") }}</ion-label>
+            <ion-label slot="end">Cyrill</ion-label>
+          </ion-item>
+          <ion-item lines="full">
+            <ion-label>{{ $t("OMS") }}</ion-label>
```
Comment (dt2patel) [inline src/views/Settings.vue:22]: This was functional before so we should keep it functional now too.

File: src/views/Settings.vue [hunk 5]
```diff
@@ -2,60 +2,123 @@
...
-        </ion-select>
-      </ion-item>
+      <div>
+        <ion-card>
+          <ion-item lines="none">
+            <ion-icon :icon="personCircleOutline" slot="start" />
+            <ion-label>{{ $t("Profile") }}</ion-label>
+          </ion-item>
+          <ion-item>
+            <ion-label>{{ $t("User") }}</ion-label>
```
Comment (dt2patel) [inline src/views/Settings.vue]: User was functional before sho it should remain functional here. User data should stay dynamic.

**Review by dt2patel — CHANGES_REQUESTED** [review]

**Review by dt2patel — APPROVED** [review]

File: src/views/Settings.vue (hunk 1 above)
Comment (azkyakhan) [reply src/views/Settings.vue]: I have removed it sir

File: src/views/Settings.vue (hunk 3 above)
Comment (azkyakhan) [reply src/views/Settings.vue]: Replaced div tag with main tag

File: src/views/Settings.vue (hunk 5 above)
Comment (azkyakhan) [reply src/views/Settings.vue]: Ok sir I have made changes.

**Pull request merged**